server/Dockerfile

```diff
@@ -109,8 +109,10 @@ RUN ./configure-apt.sh && \
     apt-get autoremove -yqq && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    rm install-ffmpeg.sh
+    rm install-ffmpeg.sh && \
+    mkdir -p /build && \
+    mv ./build-lock.json /build
```

Comment:
I am wondering if it would make more sense for the WORKDIR to be `/build` up to this point and then set to `/usr/src/app` at the very end. We should just copy the build-lock.json file and do all the configure stuff in `/build` instead of `/usr/src/app`, no?

Reply:
That makes sense